From 046deafc60a31f485a96773dc5ea5b0eb2ed15f1 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 24 Feb 2020 16:24:33 +0000
Subject: [PATCH] IWYU: missing include for std::string in csp_source_list.h

Build error in GCC:
../../services/network/public/cpp/content_security_policy/csp_source_list.h:19:6: error: ‘string’ in namespace ‘std’ does not name a type

Bug: 819294
Change-Id: I9c8665050cae038530a79323b65932a124adf2a1
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2067074
Commit-Queue: Lucas Gadani <lfg@chromium.org>
Reviewed-by: Lucas Gadani <lfg@chromium.org>
Cr-Commit-Position: refs/heads/master@{#743946}
---

diff --git a/services/network/public/cpp/content_security_policy/csp_source_list.h b/services/network/public/cpp/content_security_policy/csp_source_list.h
index 0cd3971..6df1891 100644
--- a/services/network/public/cpp/content_security_policy/csp_source_list.h
+++ b/services/network/public/cpp/content_security_policy/csp_source_list.h
@@ -5,6 +5,7 @@
 #ifndef SERVICES_NETWORK_PUBLIC_CPP_CONTENT_SECURITY_POLICY_CSP_SOURCE_LIST_H_
 #define SERVICES_NETWORK_PUBLIC_CPP_CONTENT_SECURITY_POLICY_CSP_SOURCE_LIST_H_
 
+#include <string>
 #include <vector>
 
 #include "base/component_export.h"
